lib/logstash/outputs/opensearch/http_client.rb

```diff
@@ -324,6 +324,12 @@ def build_adapter(options)
 
       adapter_options[:headers] = client_settings[:headers] if client_settings[:headers]
 
+      adapter_options[:auth_type] = options[:auth_type]
```

Comment:
These comments should be updated: https://github.com/naveentatikonda/logstash-output-opensearch/blob/IAM_Authentication/lib/logstash/outputs/opensearch/http_client.rb#L41-L49?

Reply:
updated